### PR TITLE
fix: client profile metadata request

### PR DESCRIPTION
**Pull request by krzysu**

### CODE REVIEW

File: packages/client/src/submodules/feed/graphql/feed.graphql [hunk 1]
```diff
@@ -41,7 +41,7 @@ query Feed(
   $profilePictureTransform: ImageTransform!
   $profileStatsArg: ProfileStatsArg
   $profileStatsCountOpenActionArgs: ProfileStatsCountOpenActionArgs
-  $profileMetadataSource: AppId = null
```
Comment (krzysu) [inline packages/client/src/submodules/feed/graphql/feed.graphql:44]: the issue was defaulting to null, and API expects undefined, seems null was still processed as a valid AppId returning always fallback metadata.

Comment (krzysu) [reply packages/client/src/submodules/feed/graphql/feed.graphql:44]: does it make sense?

Comment (krzysu) [reply packages/client/src/submodules/feed/graphql/feed.graphql:44]: I tested it with the same scenario as reported, having a profile with metadata, updating it, and requesting the profile again right after the update is indexed, and the issue is fixed with the code change above.

Comment (cesarenaldi) [reply packages/client/src/submodules/feed/graphql/feed.graphql:44]: I don't doubt it it's like you described it. Odd for the API to make a distinction between `null` and `undefined` as these are often interchangeable in GQL.

**Pull request merged**